stl/inc/__msvc_int128.hpp

```diff
@@ -946,11 +967,11 @@ class numeric_limits<_Unsigned128> : public _Num_int_base {
     static constexpr int digits10   = 38;
 };
 
-template <integral _Ty>
+template <class _Ty>
 struct common_type<_Ty, _Unsigned128> {
     using type = _Unsigned128;
 };

```

Comment:
I think we should equivalently provide common types when concepts are unavailable. (ditto for other `common_type` partial specializations)
```suggestion
#ifdef __cpp_lib_concepts
template <integral _Ty>
struct common_type<_Ty, _Unsigned128> {
    using type = _Unsigned128;
};
#else
template <class _Ty>
struct common_type<_Ty, _Unsigned128> : enable_if<is_integral_v<_Ty>, _Unsigned128> {};
#endif // __cpp_lib_concepts
```